index.bs

```diff
@@ -338,42 +338,61 @@ refer to [[WebAuthnAPIGuide]] for an overall tutorial.
 
 ## Use Cases ## {#sctn-use-cases}
 
-The below use case scenarios illustrate use of two very different types of [=authenticators=], as well as outline further
-scenarios. Additional scenarios, including sample code, are given later in [[#sctn-sample-scenarios]].
+Several user experiences can be crafted using this specification. This section enumerates the most common and specifies the values for the numerous options in {{PublicKeyCredentialCreationOptions}} and {{PublicKeyCredentialRequestOptions}} that will invoke them. By implementing one of these patterns, a Relying Party should find that users suffer less confusion as the structure of the interaction should be more familiar to them.
```

Comment:
rather than jumping directly into API options settings and named flows with little real-world context, I am thinking that we might largely retain the present [Use Cases section](https://w3c.github.io/webauthn/#sctn-use-cases), though appropriately edit it (move the passwordless/Bluetooth example to end, add a distinct 2nd factor scenario(?)), and incorporate links to further sections that describe the options settings yielding said use case scenarios. WDYT?